themes/ffrn/layouts/partials/footer.html

```diff
@@ -1,49 +1,51 @@
+                <a href="/impressum/">Impressum</a>
+                <hr>
+                <address>
+                    <strong>Freifunk Rhein-Neckar e.V.</strong><br>
+                    c/o Jakob Haufe<br>
+                    Hammelbächerstraße 35<br>
+                    D-69469 Weinheim<br>
+                </address>
+                <hr>
+            </div>
+            <div class="col-sm-4 footer-links">
+                <h4>Links</h4>
+                <hr>
+                <ul>
+                    <li><a target="_blank" href="https://freifunk-rhein-neckar.github.io/ffrn/mitgliedsantrag/Mitgliedsantrag.pdf">Mitgliedsantrag</a></li>
+                    <li><a target="_blank" href="https://freifunk-rhein-neckar.github.io/ffrn/satzung/satzung_ffrn.pdf">Vereinssatzung</a></li>
+                    <li><a target="_blank" href="https://freifunk-rhein-neckar.github.io/ffrn/beitragsordnung/beitragsordnung_ffrn.pdf">Beitragsordnung</a></li>
+                    <li><a target="_blank" href="http://map.freifunk-rhein-neckar.de/">Map</a></li>
+                    <li><a target="_blank" href="https://wiki.ffrn.de">Wiki</a></li>
+                    <li><a target="_blank" href="http://fw.freifunk-rhein-neckar.de">Firmware Download</a></li>
+                    <li><a target="_blank" href="https://register.freifunk-rhein-neckar.de/">Knotenverwaltung</a></li>
+                </ul>
+            </div>
+            <div class="col-sm-4 footer-links footer-social">
+                <h4>Social & Kontakt</h4>
+                <hr>
+                <ul>
+                    <li><a target="_blank" href="https://forum.ffrn.de/"><i class="fab fa-discourse"></i> Forum</a></li>
+                    <li><a target="_blank" href="https://matrix.to/#/#ffrn:ffrn.de"><i class="fa fa-comments"></i> #ffrn:ffrn.de</a> (<a href="http://chat.ffrn.de/">Matrix-Chat</a>)</li>
+                    <li><a target="_blank" href="https://twitter.com/FFRheinNeckar"><i class="fab fa-twitter"></i> Twitter</a></li>
+                    <li><a target="_blank" href="https://chaos.social/@ffrn" rel="me"><i class="fab fa-mastodon"></i> Mastodon</a></li>
+                    <li><a target="_blank" href="https://github.com/Freifunk-Rhein-Neckar"><i class="fab fa-github"></i> GitHub</a></li>
+                </ul>
+            </div>
         </div>
     </div>
 </footer>
-<script src="/js/jquery.min.js"></script>
-<script src="/js/bootstrap.min.js"></script>
-<script src="/js/headroom.min.js"></script>
-<script src="/js/headroom.conf.js"></script>
-<script type="text/javascript" src="/js/app.js"></script>
-<script type="text/javascript" src="/js/View/App.Helper.js"></script>
-<script src="/js/View/map.js"></script>
+<!-- <script src="/js/jquery.min.js"></script> -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW" crossorigin="anonymous"></script>
```

Comment:
Am besten lokal auslagern oder? Schicken wir Clients nicht in die USA ohne Zustimmung und shared caching in Browsern dürfte eh mittlerweile Geschichte sein.

Reply:
Klar, da ging es nur um ein schnelles testen. Unter `/static/` ist auch noch das ganze alte Zeug usw. das ist noch ein sehr früher Draft. Wenn man das aber im Repo hinzufügt wird der PR sehr unübersichtlich. Daher habe ich das erstmal so gemacht.